build.gradle

```diff
@@ -439,12 +439,12 @@ void mapJar(File output, File input, File mappings, File libraries, String from,
 	try {
 		def outputConsumer = new OutputConsumerPath(output.toPath())
 		outputConsumer.addNonClassFiles(input.toPath())
-		remapper.read(input.toPath())
+		remapper.readInputs(input.toPath())
 
 		libraries.eachFileRecurse(FileType.FILES) {file ->
-			remapper.read(file.toPath())
+			remapper.readInputs(file.toPath())
```

Comment:
Libraries should use readClassPath, *not* readInputs - readInputs is only for remapping inputs.

Reply:
ah, that's why cfr decompiled all those fastutil things on my built jar! didn't realize that, thanks!